woocommerce/wc-admin
# Filters: Add query from config directly to url

Depends on #591 

Some Filters simply add parameters, such as `top_sales`. These parameters have meaning for the Table, but not necessarily for the chart. For instance `orderby=gross_revenue` wreaks havoc on the Chart because it expects a chronological order:

![screen shot 2018-10-26 at 2 26 04 pm](https://user-images.githubusercontent.com/1922453/47539885-0bd50400-d92f-11e8-9713-1ac23fc82b85.png)

Selection of these filters should just update the url and let components handle what to do with the `query` object.

## Test

1. `/wp-admin/admin.php?page=wc-admin#/analytics/products`.
2. Choose filters "Top products by items sold" or "Top products by gross revenue".
3. See the Table update and the chart stay the same.
